src/elements/Progressbar/style.scss

```diff
@@ -0,0 +1,22 @@
+:local {
+  @import '../../styles/config';
+  $progress-bar-height: 13px;
+  $bar-background-color: #0290d7;
+  $container-background-color: #e6e6e9;
+  $progress-bar-radius: 30px;
+
+  .progress-bar-container {
+    background-color: $container-background-color;
+    border-radius: $progress-bar-radius;
+  }
+
+  .progress-bar {
+    text-align: center;
+    font-size: $progress-bar-height - 1px;
+    color: #fff;
```

Comment:
let's not use hex for common color. There is `$color-white` in config.scss file which is `#fff`